dc_make/maker.py

```diff
@@ -116,8 +128,54 @@ def getInputFile(self, era, model_params):
       self.input_files[file_name] = file
     return file_name, self.input_files[file_name]
 
+  def getRelevantBins(self, era, channel, category,signal_processes_histograms,unc_name=None, unc_scale=None, model_params=None):
+    relevant_bins = []
+    for hist in signal_processes_histograms:
+        axis = hist.GetXaxis()
+        hist_integral = hist.Integral(1,axis.GetNbins() + 1)
+        for nbin in range(1,axis.GetNbins() + 1):
+          isImportant=False
+          if hist_integral !=0 and hist.GetBinContent(nbin) / hist_integral >= 0.2:
+            isImportant = True
+          if len(relevant_bins) < axis.GetNbins():
+            relevant_bins.append(isImportant)
+          else:
+            relevant_bins[nbin-1] = isImportant
+    return relevant_bins
+
+  def getMultiValueLnUnc(self,unc,unc_name, process, era, channel, category, model_params):#, unc_name=None, unc_scale=None)
+    file_name, file = self.getInputFile(era, model_params)
+    hist_name = f"{channel}/{category}/{process.hist_name}"
+    if unc.getUncertaintyForProcess(process.name) != None:
+      return unc.getUncertaintyForProcess(process.name)
+    elif process.subprocesses:
+      #print(f"process has {process.subprocesses}")
+      unc_value_tot = 0.
+      yield_value_tot = 1.
+      for subp in process.subprocesses:
+        hist_name = f"{channel}/{category}/{subp}"
+        subhist = file.Get(hist_name)
+        #newhist = self.hist_binner.applyBinning(era, channel, category, model_params, subhist)
+        if subhist == None:
+          raise RuntimeError(f"Cannot find histogram {hist_name} in {file.GetName()}")
+        axis = subhist.GetXaxis()
+        yield_subproc = subhist.Integral(1,axis.GetNbins() + 1)
+        unc_value = unc.getUncertaintyForProcess(subp)
+        #print(f"got {unc_name} for {subp} with yield {yield_subproc} and it's {unc_value}")
+        if unc_value != None:
+          if yield_subproc == 0 : continue
+          unc_value_tot+=unc_value*yield_subproc
+          yield_value_tot*=yield_subproc
```

Comment:
why multiplication?

Reply:
#  uAB = ( uA * A + uB * B ) / AB

Reply:
yes, "AB" is the name of the process.... Call it C if you want...
```
C = A+B
uC = (uA * A + uB * B) / C
```